docs/reverse_proxy.md

```diff
@@ -108,6 +108,7 @@ example.com:8448 {
 
```

Comment:
while you're here: there's a spurious `;` on the line above. Can you fix that?

Reply:
sure